src/app/features/genes/components/gene-evidence-rna/gene-evidence-rna.component.ts

```diff
@@ -52,6 +56,8 @@ export class GeneEvidenceRnaComponent implements AfterViewInit {
     this.differentialExpressionYAxisMax = undefined;
 
     this.consistencyOfChangeChartData = undefined;
+
+    this.hasScrolled = false;
```

Comment:
since this variable is already initialized, perhaps this assignment can be removed.

Reply:
The `reset` method resets the other member variables to their initial value whenever the gene is changed, so I followed the same pattern for the `hasScrolled` variable. However, if `hasScrolled` shouldn't be reset when the gene changes, then I'm happy to remove this assignment!

Reply:
ah, makes sense